public/audioplayer.html

```diff
@@ -11,6 +11,18 @@
         document.getElementById("myScale").innerHTML = `${queryObject.key} ${queryObject.scale}`
     }
     
+    async function getNewMelody() {
+        const url = `${window.location.origin}/.netlify/functions/server`
+        const response = await fetchAsync(url)
+        window.location.href = response.url
+    }
+
+    async function fetchAsync(url) {
+        let response = await fetch(url);
+        let data = await response.json();
```

Comment:
since these variables aren't changing
```suggestion
        const response = await fetch(url);
        const data = await response.json();
```